addons/addons.json

```diff
@@ -14,6 +14,7 @@
   "// For website:",
   "scratchr2",
   "dark-www",
+  "message-filters",
```

Comment:
This meant themes for the website. Bring this to the bottom of the list.

Reply:
Got it